src/components/ContentListTable/ContentListTable.tsx

```diff
@@ -51,6 +46,11 @@ const ContentListTable = () => {
   const [page, setPage] = useState(1);
   const [perPage, setPerPage] = useState(storedPerPage);
 
+  const {
+    debouncedSearchQuery,
+    labels
```

Comment:
```suggestion
   filterData
```

Reply:
![Screen Shot 2022-07-19 at 10 04 45 AM](https://user-images.githubusercontent.com/38083295/179797197-577ef800-57d9-4536-a89b-b4ad28e1b90e.png)

Could we change this to filterData? As this is only used for the API we can keep the idea of it similar to within the API.